src/SFA.DAS.Campaign.Domain/Interfaces/ICacheStorageService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.Campaign.Domain.Interfaces
+{
+    public interface ICacheStorageService
```

Comment:
Don't think this should live in the Domain. Would say it's probably an application or even infrastructure interface.